### DISCUSSION TITLE
Add `configPlugin` option to `@nx/rollup:rollup` executor

### DISCUSSION THREAD

**Discussion by oliveryasuna** [nrwl/nx#22717] [Feature Requests]
Not sure how often this feature is utilized, but I use the Rollup's `--configPlugin` option to specify [@rollup/plugin-typescript](https://www.npmjs.com/package/@rollup/plugin-typescript) in order to write my Rollup config in TypeScript.

Either `--configPlugin typescript` or `--configPlugin @rollup/plugin-typescript`.

It would be useful if the executor supported an option named `configPlugin`.

For now, I will write my Rollup configs in JavaScript and use JSDoc type hints.

**Comment by dfbaskin** [comment]
This was helpful discussion about @rollup/plugin-typescript vs rollup-plugin-typescript2.  
https://github.com/rollup/plugins/issues/541
